mautrix_telegram/__main__.py

```diff
@@ -101,7 +103,10 @@ def prepare_bridge(self) -> None:
             self.add_startup_actions(self.bot.start())
         if self.config["bridge.resend_bridge_info"]:
             self.add_startup_actions(self.resend_bridge_info())
-        self.add_startup_actions(self._loop_active_puppet_metric())
+
+        # Do not block startup on this
+        self.periodic_sync_task = asyncio.create_task(self._loop_active_puppet_metric())
```

Comment:
Should this execute somewhere else?

Reply:
I can't think of anything more fitting. I thought that `periodic_active_metrics_task` would be a prior art to draw some inspiration from, but it turns out to be... an unused attribute.

I'd maybe drop the comment, it's an explanation for the PR in context, but not so much for the following line :) Or maybe note something like "and this is why we're not making this a startup action", or something.